Project-Hotpot/Project-Hotpot/Algorithm/Data Structures/SongDetails.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  SongDetails.swift
+//  Project-Hotpot
+//
+//  Created by Harini Sundaram on 7/27/22.
+//
+
+import Foundation
+
+class SongDetails: NSObject{
```

Comment:
Consider:
1. Using a struct instead of a class. Structs contain an `init` for you so you don't have to write it separately.
2. Define `SongDetails` as `Codable` (https://developer.apple.com/documentation/swift/codable). Codable is most typically used to auto-populate objects from JSON, but you can use this to auto-convert from/to just about anything you're interested in.